tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Test_KerrSchild.cpp

```diff
@@ -0,0 +1,187 @@
+  // g_{ij}   = delta_{ij} + 2 M x_i x_j/r^3
+  // d_i H    = -Mx_i/r^3
+  // d_i l_j  = delta_{ij}/r - x^i x^j/r^3
+  // d_k g_ij = -2M x_i x_j x_k/r^5 + 4(M/r)x_(i/r [delta_{kj)}/r-x_k x_j)/r^3]
+  //          = -6M x_i x_j x_k/r^5 + 2 M x_i delta_{kj}/r^3
+  //                                + 2 M x_j delta_{ki}/r^3
+
+  // Parameters
+  //   ...for KerrSchild solution
+  const double mass = 1.01;
+  const std::array<double, 3> spin = {{0.0, 0.0, 0.0}};
+  const std::array<double, 3> center = {{0.0, 0.0, 0.0}};
+  //   ...for grid
+  const size_t grid_size = 3;
+  const std::array<double, 3> lower_bound = {{1.99, -0.01, -0.01}};
+  const std::array<double, 3> upper_bound = {{2.01, 0.01, 0.01}};
+
+  // Set up grid
+  const size_t data_size = pow<3>(grid_size);
+  Index<3> extents(grid_size);
+
+  // Set up map
+  CoordinateMaps::AffineMap x_map(-1., 1., lower_bound[0], upper_bound[0]);
+  CoordinateMaps::AffineMap y_map(-1., 1., lower_bound[1], upper_bound[1]);
+  CoordinateMaps::AffineMap z_map(-1., 1., lower_bound[2], upper_bound[2]);
+  CoordinateMaps::ProductOf3Maps<CoordinateMaps::AffineMap,
+                                 CoordinateMaps::AffineMap,
+                                 CoordinateMaps::AffineMap>
+      xyz_map(x_map, y_map, z_map);
+
+  CoordinateMap<Frame::Logical, Frame::Inertial,
+                CoordinateMaps::ProductOf3Maps<CoordinateMaps::AffineMap,
+                                               CoordinateMaps::AffineMap,
+                                               CoordinateMaps::AffineMap>>
+      coord_map(xyz_map);
+
+  // Set up coordinates
+  const auto x_logical = logical_coordinates(extents);
+  const auto x = coord_map(x_logical);
+  const double t = 1.3;
+
+  // Evaluate solution
+  EinsteinSolutions::KerrSchild solution(mass, spin, center);
+
+  const auto vars = solution.solution(x, t);
+  const auto& lapse = get<EinsteinSolutions::KerrSchild::lapse>(vars);
+  const auto& dt_lapse = get<EinsteinSolutions::KerrSchild::dt_lapse>(vars);
+  const auto& d_lapse = get<EinsteinSolutions::KerrSchild::deriv_lapse>(vars);
+  const auto& shift = get<EinsteinSolutions::KerrSchild::shift>(vars);
+  const auto& d_shift = get<EinsteinSolutions::KerrSchild::deriv_shift>(vars);
+  const auto& dt_shift = get<EinsteinSolutions::KerrSchild::dt_shift>(vars);
+  const auto& g = get<EinsteinSolutions::KerrSchild::spatial_metric>(vars);
+  const auto& dt_g =
+      get<EinsteinSolutions::KerrSchild::dt_spatial_metric>(vars);
+  const auto& d_g =
+      get<EinsteinSolutions::KerrSchild::deriv_spatial_metric>(vars);
+
+  // Check those quantities that should be zero.
+  const auto zero = make_with_value<DataVector>(x, 0.);
+  CHECK(dt_lapse.get() == zero);
+  for (size_t i = 0; i < 3; ++i) {
+    CHECK(dt_shift.get(i) == zero);
+    for (size_t j = 0; j < 3; ++j) {
+      CHECK(dt_g.get(i, j) == zero);
+    }
+  }
+
+  const DataVector r = magnitude(x);
+  auto lapse_test = make_with_value<Scalar<DataVector>>(x, 0.0);
```

Comment:
I think appending `_test` doesn't make it clear whether this the expected lapse or the computed. I prefer to prepend `expected_`, I don't feel too strongly though, what do other people think?